mitodl/edx-platform
# Revert "#545 Allow user to create video using HLS link"

This reverts commit 1068297eee33b85f0a0586f850d8db4882573c65 (PR #138), which added a UI to add HLS videos in Studio

### To test:

- Perform the configuration steps mentioned in [this comment](https://github.com/mitodl/edx-platform/pull/138#issuecomment-509363890)
- Visit the "Video Uploads" page (in the "Content" menu; example URL: http://edx.odl.local:18010/videos/course-v1:MIT+DemoX+Demo_Course_2)

The page should load, and the UI should not have the section show in [this screenshot](https://github.com/mitodl/edx-platform/pull/138#issue-294916239)
